src/modeler/AutoRest.Swagger/Model/ValidationUtilities.cs

```diff
@@ -262,20 +262,23 @@ public static bool IsArrayTypeResponseOperation(Operation op, ServiceDefinition
             if (op.Responses["200"]?.Schema?.Reference?.Equals(string.Empty) == false)
             {
                 var modelLink = op.Responses["200"].Schema.Reference;
+
```

Comment:
The issue exists (#2082) for more than three weeks now and note that this is NOT just a regular linter rule! That rule would have to magically be enforced to run first and it would have to prevent execution of further ones (or the other ones are simply made robust, as this PR tries to do).